xhprof_lib/utils/xhprof_runs.php

```diff
@@ -107,22 +107,22 @@ public static function getDbClass()
   * 
   *
 CREATE TABLE `details` (
```

Comment:
as for the size of data columns, id ok was a wrong one but was when I was debugging because data was not being inserted due to the server_id only allowing 3 chars, which personally doesn't make any sense and might limit users, for instance, I use AWS , so for the server_id i'd use the instance-id, which is way longer than 3 chars.

As for the lowercase changing, blame mysql cli, show create table details;

I will revert the id back to 17, but will keep the rest.